src/BladewindServiceProvider.php

```diff
@@ -6,20 +6,26 @@
 
 class BladewindServiceProvider extends ServiceProvider
 {
-  public function boot()
-  {
-    $this->loadTranslationsFrom(__DIR__ . '/../lang', 'bladewind');
-    
-    $this->loadViewsFrom(__DIR__ . '/../resources/views', 'bladewind');
-
-    $this->publishes([
-        __DIR__ . '/../resources/views/components/' => resource_path('views/components/bladewind'),
-        __DIR__ . '/../resources/lang' => function_exists('lang_path') ? lang_path() : resource_path('lang'),
-        __DIR__ . '/../resources/assets/compiled' => public_path('bladewind'),
-    ], 'bladewind-assets'); // laravel-assets: changed to bladewind-assets to prevent automatic publishing on composer update
-
-    $this->publishes([
-        __DIR__.'/resources/assets/raw' => resource_path('bladewind'),
-    ], 'raw-css');
-  }
-}
\ No newline at end of file
+    public function boot()
+    {
+        $this->loadTranslationsFrom(__DIR__ . '/../lang', 'bladewind');
+
+        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'bladewind');
+
+        $this->publishes([
```

Comment:
The user doesn't need to publish the components, assets and language files in 3 different steps because the assets are required for the components to work. For example, without the language files, the date picker will not display properly

Reply:
Should we add register component?, maybe some user don’t need publish all resources

Reply:
Publishing `bladewind-components` should publish `bladewind-lang` at least. This will  work for majority of users. The rest can be optional as you have it. I will merge and make the change.